src/main/java/az/company/mssubscription/controller/SubscriptionController.java

```diff
@@ -0,0 +1,30 @@
+package az.company.mssubscription.controller;
+
+import az.company.mssubscription.model.request.CreateSubscriptionRequest;
+import az.company.mssubscription.service.SubscriptionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import static org.springframework.http.HttpStatus.CREATED;
+
+@RestController
+@RequestMapping("/v1/subscriptions")
+@RequiredArgsConstructor
+public class SubscriptionController {
+
+    private final SubscriptionService subscriptionService;
+
+    @PostMapping
+    @ResponseStatus(CREATED)
+    public void createSubscription(@RequestBody CreateSubscriptionRequest createSubscriptionRequest){
+        subscriptionService.createSubscription(createSubscriptionRequest);
+    }
+
+    @PutMapping("/cancel-subscription/{id}")
```

Comment:
1. invalid rest api naming convention, /v1/subscriptions/{id}/cancel 
2. [Optional] - you can change PUT to PATCH
3. use static import everywhere this is possible